adapters/oidc/adapter-core/src/main/java/org/keycloak/adapters/HttpClientBuilder.java

```diff
@@ -113,9 +113,9 @@ public void checkServerTrusted(X509Certificate[] chain,
     protected long connectionTTL = -1;
     protected TimeUnit connectionTTLUnit = TimeUnit.MILLISECONDS;
     protected HostnameVerifier verifier = null;
-    protected long socketTimeout = -1;
+    protected int socketTimeout = -1;
```

Comment:
Why this change?

Reply:
I've been discussing it with @mhajas and considering to change from _long_ to _int_ in order to follow Apache HTTP client API. However, as you've mentioned, this is unrelated to the JIRA and the change is not necessary though.

Reply:
Please keep this PR minimal, it is already big enough.